### PR TITLE
add reducers and actions

### PR DESCRIPTION
**Pull request by Owusu-Desmond**
In this pull request, I make the following changes

- [ ] Install redux.
- [ ] Create reducers and actions for books
- [ ] Create reducers and actions for categories.
- [ ] Create a config function for store.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+});
+
+const removeBook = () => ({
+  type: '',
+  payload: {
+    title: 'Harry Potter',
+  },
+});
+
+export { addBook, removeBook };
```
Comment (mikethreels) [inline src/redux/books/books.js]: - [ ] in this file you should Define ADD_BOOK and REMOVE_BOOK as action types, please keep in mind that The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux)

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  switch (action.type) {
+    case 'CHECK_STATUS':
+      return 'Under construction';
+    default:
+      return state;
+  }
+};
+
+export { checkStatus };
+export default categoriesReducer;
```
Comment (mikethreels) [inline src/redux/categories/categories.js]: - [ ] in this file you should Define CHECK_STATUS as action type, please keep in mind that The value of action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux)

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Desmond, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Owusu-Desmond, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean/Organized code

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**